### DISCUSSION TITLE
Did the hotkeys change?

### DISCUSSION THREAD

**Discussion by bruno-brant** [freeplane/freeplane#554] [Help] [answered]
Sorry for the seemly crazy question, but, did some hotkeys change "recently"? I'm under the impression that I used `insert` to add nodes, not tab.

This is not that relevant, since I can just reassign keys (and, in fact, I prefer tab to insert), but I'm curious... I manage my installation of Freeplane through scoop.sh, and I was probably late to the latest version for a while...

**Comment by macmarrum** [comment, marked as answer]
Yes, they did. Please see also #493
